conda.recipe/post-link.bat

```diff
@@ -1 +1 @@
-"%PREFIX%\bin\python" -m nb_conda_kernels.install --enable --prefix="%PREFIX%"
+"%PREFIX%\python" -m nb_conda_kernels.install --enable --prefix="%PREFIX%"
```

Comment:
You mean:

`"%PYTHON%" -m nb_conda_kernels.install --enable --prefix="%PREFIX%"`

I am OK with that...